internal/uid/uid.go

```diff
@@ -0,0 +1,16 @@
+package uid
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetRootlessUID returns the UID of the user in the parent userNS
+func GetRootlessUID() int {
```

Comment:
This should all talk about EUID, in all of function/variable names and comments, because that’s what the value apparently is (except the environment variable must not be renamed).